.travis.yml

```diff
@@ -2,5 +2,7 @@ language: node_js
 
 script: npm test && npm run integration-tests
 
+after_script: .vinyldns/bin/remove-vinyl-containers.sh
```

Comment:
Don't think this is needed since `bin/run-integration-tests.sh` includes `.vinyldns/bin/remove-vinyl-containers.sh` at the end. Confirmed that running `script` locally did not leave any containers running.

Reply:
Yep - I believe you're correct. I added this as a sanity check to ensure that, in the case of a failure, a re-run of a Travis CI build won't recycle a previously-used "environment" that has a running, not-a-clean-room VinylDNS API. To be fair, though, I don't know if Travis CI recycles "environments" like that.